capellambse/model/crosslayer/fa.py

```diff
@@ -150,10 +150,39 @@ def owner(self) -> ComponentExchange | None:
         return self.allocating_component_exchange
 
 
+@c.xtype_handler(None)
+class FunctionalChainInvolvementLink(c.GenericElement):
+    """An element linking a FunctionalChain to an Exchange."""
+
+    exchanged_items = c.AttrProxyAccessor(
+        information.ExchangeItem, "exchangedItems", aslist=c.ElementList
+    )
+    exchange_context = c.AttrProxyAccessor(
+        capellacore.Constraint, "exchangeContext"
+    )
+    involved = c.AttrProxyAccessor(c.GenericElement, "involved")
+
+    @property
+    def name(self) -> str:  # type: ignore
+        return f"[{self.__class__.__name__}] to {self.involved.name}"
```

Comment:
Names are not necessarily unique, it might be good to show the involved's UUID here as well just to be sure. (Or you could use the new `_short_repr_()`.)